### ISSUE TITLE
Error on filenames with spaces

### ISSUE THREAD

**Issue by Dark-Fx** [thomasmoelhave/tcx-tools#4]
Greetings,

There is an issue with how you are handling filenames that have spaces when you are passing them as an argument to saxon-xslt. The correction should be to encapsulate the filenames like so:

saxon-xslt -o "${output}" "${input}" tcx-strip-heart-rate.xsl

Regards,
Brian McPherson

**Comment by thomasmoelhave** [comment]
Yeah, that makes sense. If you make a pull request I'll merge it :).
